test/unit/test_disco_elasticache.py

```diff
@@ -64,7 +64,8 @@ def setUp(self):
                 'port': '1000',
                 'parameter_group': 'default',
                 'num_nodes': '5',
-                'auto_failover': 'true'
+                'auto_failover': 'true',
+                'maintenance_window': 'sun:10:00-sun:11:00'
```

Comment:
The test code doesnt test the scenario when `maintenance_window` is not provided.